owl/EASE-ACT.owl

```diff
@@ -353,28 +391,64 @@ Deduction is often explained as starting from the &quot;general&quot; (some prop
     <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#Dropping -->
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#Dropping">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#PhysicalTask"/>
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#Actuating"/>
         <owl:disjointWith rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#Placing"/>
+        <ease:ELANName>move-object-drop</ease:ELANName>
+        <ease:ELANUsageGuideline>Agent undoes whatever functional control they have on an object, but without imparting significant momentum to it, and allows the object to fall under the influence of gravity.</ease:ELANUsageGuideline>
         <rdfs:comment>The dropped object falls mainly under the influence of gravity. However, an agent may also drop something during navigation. The difference to &apos;Throwing&apos; is that there is no &apos;Limb motion&apos; which is a constitiuent of the action.
 
 &apos;Dropping&apos; is intentional. Dropping by accident may not has a phase to release the grasp. It could be that the grasp was not strong enough and the objects &quot;slips&quot; away.</rdfs:comment>
     </owl:Class>
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#EndEffectorPositioning -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#EndEffectorPositioning">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#ManipulationTask"/>
+        <ease:ELANName>manipulation-hand-positioning</ease:ELANName>
+        <ease:ELANUsageGuideline>Superconcept for the Agent moving their arm(s) to positioning their hand/end effector; use more specific labels where appropriate.</ease:ELANUsageGuideline>
+    </owl:Class>
+    
+
+
     <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#Grasping -->
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#Grasping">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#PhysicalTask"/>
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#ManipulationTask"/>
         <owl:disjointWith rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#Releasing"/>
+        <ease:ELANName>manipulation-grasp</ease:ELANName>
+        <ease:ELANUsageGuideline>Agent grasps an object.</ease:ELANUsageGuideline>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#HeadMovement -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#HeadMovement">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#PhysicalTask"/>
```

Comment:
I would not say that *HeadMovement* is a task. it can also happen reflexive. It would be rather a *MotionType*.

While looking for something is a task that includes the head motion.

Reply:
Ok, I will move the relevant bits to the movement type branch.

Reply:
Also added a BodyMovement superclass for HeadMovement.